### PR TITLE
Fixes #99. Update webpack dependency to v4.

### LINKED ISSUE #99: Update webpack dependency to v4
I'll work on fixing this last dependency (partitioned from original [Issue 95](https://github.com/webcompat/webcompat-reporter-extensions/issues/95)).

### PR DESCRIPTION
**Pull request by laghee**
Not sure I should have added the webpack-cli dev dependency (I'd forgotten having used the -D flag as prompted), but tests now passing locally. 
r? @miketaylr

### CODE REVIEW

**Comment by miketaylr** [comment]
> Not sure I should have added the webpack-cli dev dependency

Yep! It was failing for me locally without having installed that. Thanks for working on this!

File: chrome/webpack.config.js [hunk 1]
```diff
@@ -10,21 +10,21 @@ const version = require("../package.json").version;
 module.exports = {
   entry: "./chrome/addon.js",
   output: {
-    filename: "./dist/chrome/background.js"
+    filename: "../dist/chrome/background.js"
```
Comment (miketaylr) [inline chrome/webpack.config.js]: I wonder *what* changed between 3 and 4 that caused this to stop working. 🤷‍♂️ 🤷‍♀️

Comment (laghee) [reply chrome/webpack.config.js]: It does seem sort of random! I tried a bunch of searches to try to figure out what was causing it, and whether it was webpack itself or the fact that webpack 4 needed the 4.5 upgrade of copy-webpack-plugin and *that* did something wonky ... but I didn't find any similar reports.

Comment (miketaylr) [reply chrome/webpack.config.js]: Thanks for checking! Let's just blame ghosts. 👻

**Pull request merged**